sdk/aot/ci.yml

```diff
@@ -37,7 +37,10 @@ extends:
       - name: azure-aot-graalvm-support
         groupId: com.azure
         safeName: azureaotgraalvmsupport
+        skipPublishDocMs: true
```

Comment:
@srnagar Please suggest whether we want the skip config. I can remove this if we still want to have the publish Docs Ms step in release pipeline.

Reply:
We should not skip the publishing step in the release pipeline. We should continue without failing the build if no docs are found. This will allow us to automatically publish when we do have JavaDocs in the future.